src/ophys_etl/pipelines/dff.py

```diff
@@ -112,6 +119,16 @@ def filter_s_to_frames(self, item, **kwargs):
         return item
 
 
+def job_call(index: int, input_file: Path, key: str,
```

Comment:
How is the memory usage of this with a lot of parallel jobs?

Reply:
it's about 3G per worker